### ISSUE TITLE
Godot takes a long time to open and run any project even an empty one

### ISSUE THREAD

**Issue by SOSeriously** [godotengine/godot#84300]
### Godot version

4.1.2.stable.official [399c9dc39]

### System information

Godot v4.1.2.stable - Windows 11 pro version 10.0.22621 - Vulkan API 1.3.260 (Mobile) - dedicated NVIDIA GeForce GTX 1080 (NVIDIA; 31.0.15.4601) - AMD Ryzen 5 3500 6-Core Processor (6 Threads)

### Issue description

When opening Godot 4 it displays a blank window for around 30 to 40 seconds before showing the Godot logo and same thing when running any project even a blank project. I have managed to get it working temporarily by deleting the Godot folders in appdata located at C:\Users\{username}\AppData\Roaming\Godot and C:\Users\{username}\AppData\Local\Godot and then restart the pc (just deleting the folders will not work). Godot then works for a few hours before doing the same again. 

### Steps to reproduce

I haven't found exactly what causes it. Only that the issue returns after using Godot 4 for an hour or so on any project.

### Minimal reproduction project

N/A

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot/issues/20566.

Do you have any controllers or other USB devices connected? Check the linked issue for known problematic devices/software.

**Comment by SOSeriously** [comment]
The only usb devices connected:
Keyboard: 
coolermaster devastator 3 (no software)

Mouse: 
corsair harpoon rgb pro. I have icue installed and is up to date. The firmware is v3.11 as v3.12 and 3.13 has a double click bug.

Headset:
Logitech g633. I have Logitech G Hub installed and up to date. The firmware is also up to date.

I did run windows disk cleanup and uninstalled hidden devices in device manager. Nvidia drivers are up to date. I did run SFC, DISM and CHKDSK commands for good measure. 

Will need to test if that fixed it.
